docs/pages/blog/turbo-1-4-0.mdx

```diff
@@ -0,0 +1,134 @@
+---
+title: Turborepo 1.4
+date: 2022/07/28
+description: Turborepo 1.4 introduces the Turborepo Agent, has new examples, automatically hashes environment variables, and more!
+tag: web development
+ogImage: /images/blog/turbo-1-4-0/og-image.png
+---
+
+# Turborepo 1.4
+
+import { Authors } from '../../components/Authors'
+import Callout from '../../components/Callout'
```

Comment:
Is this blog authors? Or contributors to the content? I always thought it was the latter based on the previous posts. 

I would argue _this_ is the correct way to do it though.

Reply:
It's been kinda unclear, but I'm going with post authors here. Contributors to the release have a spot at the bottom.